everviz/pages/objectives.py

```diff
@@ -92,6 +92,13 @@ def _set_up_data_sources(api):
     )
 
 
+def _single_objective_title(api):
+    no_function = len(api.objective_function_names)
+    if no_function > 1:
```

Comment:
This is very nitpicky, but can you rename this to `nr_functions` or something or that nature? I at least read `no function`

Reply:
It is not nitpicky at all, I wanted to change it just forgot to do it.